addons/l10n_de/models/res_config_settings.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, api, _
+from odoo.exceptions import ValidationError
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    @api.model_create_multi
+    def create(self, vals_list):
```

Comment:
grepping on ValidationError in res_config_settings.py files, two out of three are in a @api.constraints

Reply:
Yes and it feels much better